static/src/javascripts/projects/common/modules/atoms/youtube-player.js

```diff
@@ -6,11 +6,18 @@ import { loadScript } from 'lib/load-script';
 import { constructQuery } from 'lib/url';
 import { getPageTargeting } from 'common/modules/commercial/build-page-targeting';
 import { commercialFeatures } from 'common/modules/commercial/commercial-features';
-import { onIabConsentNotification } from '@guardian/consent-management-platform';
 import $ from 'lib/$';
 import { buildPfpEvent } from 'common/modules/video/ga-helper';
+
```

Comment:
Just being picky here but I would prefer to avoid leaving blank lines between imports. Same on line 14.

Reply:
```suggestion
```